schemas/IDLs/org.gel.models.participant.avro/1.2.0/RDParticipant.avdl

```diff
@@ -42,12 +43,16 @@ protocol Pedigrees {
 
     enum SpatialPattern {DISTAL, GENERALIZED, LOCALIZED, PROXIMAL}
 
-
+    /**
+    TODO: see if ageOfOnset needs to be added to HpoTermModifiders for GMS
+    */
     record HpoTermModifiers{
         union {null, Laterality} laterality;
         union {null, Progression} progression;
         union {null, Severity} severity;
         union {null, SpatialPattern} spatialPattern;
+        union {null, string} hpoModifierCode;
+        union {null, string} hpoModifierVersion;
```

Comment:
laterality, progression, severity and spatial pattern are all modifiers in HPO. I understand having `hpoModifierCode` fits adding any other HPO modifier out of the previous list, but what does `hpoModifierVersion` stand for?

Reply:
@priesgo  good spot.  This is for provenance of hpoModifier Codes with respect to the hpoModifier version.  It is a non mandatory field in Elucidata but Richard Scott has been clear there will be data populated there from the codesystem.codesystem_version corresponding to relevant observation_component_code_cid field in the Elucidata peristent data model.

Is this field very relevant?  No.  Will this field possibly be useful? Yes